src/WebApp/Pages/Questions.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@page "/{category}/questions"
+@model Codidact.Core.WebApp.Pages.Posts.QuestionsModel
+@{
+    ViewData["Title"] = "Questions";
+}
```

Comment:
Showing only posts from the currently chosen category doesn't seem to work yet. Creating a new category and choosing its URL (/meta/Questions) shows the exact same set as the main category (/main/questions)

Reply:
Are you sure you pulled? Right now accessing a category that doesn't exist throws a server exception.
I added an integration test for this too: https://github.com/misha130/core/blob/5a89747ae81698aa70b9bfb01d9035fbf325204a/tests/WebApp.IntegrationTests/Pages/QuestionsPageTests.cs#L37

Reply:
Yeah. It throws an exception, but it appears, that the question lists aren't adopted (/main/Questions == /meta/Questions). I'll try again. Hold on a second.

Reply:
Can still repro with latest version:

![Screenshot showing two browser windows with one having main and the other meta open and both show the same question list](https://user-images.githubusercontent.com/21335202/79121282-37eb7b80-7d95-11ea-9ff7-b161555d15e2.png)

Reply:
This is odd.
[![Image from Gyazo](https://i.gyazo.com/5950363cb4872f0217ed55b896e656f0.png)](https://gyazo.com/5950363cb4872f0217ed55b896e656f0)

Can you run the test to see if it fails too?

Reply:
Ah. I mean: I created a new category meta in pgAdmin, but didn't assign any post to it.

Reply:
Alright, https://github.com/codidact/core/pull/88/commits/2c4ee0543e4f4489aaf33e83604a23956dc83ed8 should resolve this

Reply:
Fix confirmed.